rear_rider_device/bluetooth.py

```diff
@@ -89,7 +93,11 @@ def is_strobe_on(self):
             pass
     
     def discoverable_changed(self, value: str):
-        self.writeline(f'discoverable\n{value}')
+        try:
+            timeout = self.rear_rider_bt.get_discoverable_timeout()
+            self.writeline(f'discoverable\n{value} {timeout}')
+        finally:
+            pass
```

Comment:
same here - can you add a little exception handler even if it just prints or bubbles up? also, if your `finally` block is just passing, you don't need it
```suggestion
        try:
            timeout = self.rear_rider_bt.get_discoverable_timeout()
            self.writeline(f'discoverable\n{value} {timeout}')
        except Exception as e:
            print(e.message)
```

Reply:
Hmmm... idk what I was trying to do here. I'll just remove it and let it bubble up.